March 2025 newsletter (#336)
ddev/ddev.com: src/content/blog/ddev-march-2025-newsletter.md

---
title: "DDEV March 2025 Newsletter"
pubDate: 2025-03-04
#modifiedDate: 2025-02-06
summary: DDEV March 2025 Newsletter
author: Randy Fay
#featureImage:
#  src: /img/blog/2025/02/nancy-lewis-winter-scene.jpg
#  alt: Nancy Lewis winter scene painting
#  credit: "Plateau Creek Mile Marker 55, by Nancy Lewis"
categories:
  - Community
---

**Happy March!**

> **DDEV v1.24.3** was [released](https://github.com/ddev/ddev/releases/tag/v1.24.3) earlier than planned for two major reasons: An upcoming Docker engine release was going to be incompatible with current Mutagen version, and the new `generic` web server type allows Node.js and other interesting possibilities. **We encourage you all to upgrade because of this upcoming incompatibility, thanks!**

Reviewer: ```suggestion
**DDEV v1.24.3** was [released](https://github.com/ddev/ddev/releases/tag/v1.24.3) earlier than planned for two major reasons: an upcoming Docker engine release was going to be incompatible with current Mutagen version, and the new `generic` web server type allows Node.js and other interesting possibilities. **We encourage you all to upgrade because of this upcoming incompatibility, thanks!**
```

Maybe consider having the reasons in a bulleted list as well:

```suggestion
**DDEV v1.24.3** was [released](https://github.com/ddev/ddev/releases/tag/v1.24.3) earlier than planned for two major reasons:

- an upcoming Docker engine release was going to be incompatible with current Mutagen version
- the new `generic` web server type allows Node.js and other interesting possibilities.

**We encourage you all to upgrade because of this upcoming incompatibility, thanks!**
```

The list makes it easier to parse - though without the list is more conversational, so really depends what kinda tone you're going for and how much you want people to grok the "two major reasons"

> **Open-Source For The Win**: See the details about how a future incompatibility with the Docker engine was pre-emptively caught and fixed, [Open Source for the Win!](https://20250303-march-newsletter.ddev-com-front-end.pages.dev/blog/open-source-for-the-win/)

Reviewer 2: ```suggestion
**Open-Source For The Win**: See the details about how a future incompatibility with the Docker engine was pre-emptively caught and fixed, [Open Source for the Win!](open-source-for-the-win.md)
```

> **Web-based DDEV Add-on Registry** is now live! Try it out [https://addons.ddev.com](https://addons.ddev.com)

Reviewer 2: ```suggestion
**Web-based DDEV Add-on Registry** is now live! Try it out at <https://addons.ddev.com> and check out the [introduction blog](ddev-addon-registry-introduction.md).
```

> **TYPO3 Community Budget Ideas: Please vote!**: We [applied for the Q2 approval process](https://talk.typo3.org/t/implement-mdns-for-ddev-name-resolution/6119/4) to fund a great feature for DDEV, supporting mDNS as a domain-name resolution technique. If you're a member of the TYPO3 Association, you've received an email to vote on this. We'd appreciate your vote!

Reviewer 2: ```suggestion
**TYPO3 Community Budget Ideas: Please vote!**: We [applied for the Q2 approval process](https://talk.typo3.org/t/implement-mdns-for-ddev-name-resolution/6119/4) to fund a great feature for DDEV, supporting mDNS as a domain-name resolution technique. If you're a member of the TYPO3 Association, you should have received an email to vote on this. We'd appreciate your vote!
```

**Platform.sh has transferred domain names to DDEV**: Thanks again to [Platform.sh](https://platform.sh) for their ongoing support. As a part of the process in [change of their support](platform-sh-ddev-funding-changes.md), they have transferred control of the `ddev.com` and `ddev.site` domain names to the DDEV Foundation.

**DDEV Notes and News**

- The DDEV Advisory Group [Annual Review and Planning Meeting](https://github.com/orgs/ddev/discussions/7031) is on Wednesday, March 5 (tomorrow!), and **all are invited**. The Zoom link is in the agenda link. We'll be looking at 2025 ambitions and 2024 review, we'd love to have you there!
- Randy presented on [Divide and conquer: A systematic approach to troubleshooting issues](https://www.fldrupal.camp/session/divide-and-conquer-systematic-approach-troubleshooting-issues) at Florida Drupalcamp. The full recording is on the link.
- [Installing Drupal CMS with DDEV](https://www.webwash.net/drupal-cms-v1-installation-and-demo/)
- Docker has renewed DDEV's "Docker-Sponsored Open Source" membership, meaning that DDEV users don't have a pull limit on DDEV images on hub.docker.com. This is a great thing for DDEV. Thanks, Docker!

> - We applied for the [Google Summer of Code](https://summerofcode.withgoogle.com/) but did not get accepted.

Reviewer 2: ```suggestion
- We applied for the [Google Summer of Code](https://summerofcode.withgoogle.com/) but did not get accepted due to the high volume of applications and limited available slots. We'll try again in the future!
```

> - We're working on integrating the excellent Xhgui performance exploration GUI for Xhprof to DDEV. This work is being funded by the TYPO3 Community's Q1 Community Budget Idea, and need to be completed in March.

Reviewer 2: ```suggestion
- We're working on integrating the excellent XHGui performance exploration GUI for Xhprof to DDEV. This work is being funded by the TYPO3 Community's Q1 Community Budget Idea, and need to be completed in March.
```

---
title: "DDEV March 2025 Newsletter"
pubDate: 2025-03-04
#modifiedDate: 2025-02-06
summary: DDEV March 2025 Newsletter
author: Randy Fay
#featureImage:
#  src: /img/blog/2025/02/nancy-lewis-winter-scene.jpg
#  alt: Nancy Lewis winter scene painting
#  credit: "Plateau Creek Mile Marker 55, by Nancy Lewis"
categories:
  - Community
---

**Happy March!**

> **DDEV v1.24.3** was [released](https://github.com/ddev/ddev/releases/tag/v1.24.3) earlier than planned for two major reasons: an upcoming Docker engine release was going to be incompatible with current Mutagen version, and the new `generic` web server type allows Node.js and other interesting possibilities. **We encourage you all to upgrade because of this upcoming incompatibility, thanks!**

Reviewer 3: ```suggestion
**DDEV v1.24.3** was [released](https://github.com/ddev/ddev/releases/tag/v1.24.3) earlier than planned for two major reasons: the new `generic` web server type allows Node.js and other interesting possibilities, and an upcoming Docker engine release was going to be incompatible with current Mutagen version. **We encourage you all to upgrade because of this upcoming incompatibility, thanks!**
```

It feels strange to flip from bug, exciting possiblies, back to bug.

Author: I can go with that. The reason it's early is the bug, but still your language is better.
